scripts/production/homology_stats_rapid.py

```diff
@@ -154,12 +157,20 @@ def main() -> None:
 
     rr_dbc = DBConnection(db_url)
 
-    ref_gdb, ref_species, *_unused = get_closest_ref(rr_dbc, ref_db)
+    ref_gdb, ref_species, _, query_species, _ = get_closest_ref(rr_dbc, ref_db)
 
     nr_homologs, nr_genes = query_rr_database(rr_dbc, ref_gdb, ref_db)
+    perc_homologs = round((nr_homologs * 100 / nr_genes), 1)
     json_data = {
```

Comment:
OK so, let's get this started:
```suggestion
    json_data: Dict[str, Any] = {
```
If you accept this suggestion, don't forget to import `Any` and `Dict` from the `typing` module.

Reply:
An alternative, more minimal approach:
```suggestion
    json_data: Dict = {
```